src/v2/providers/firestore.ts

```diff
@@ -153,6 +177,50 @@ export function onDocumentWritten<Document extends string>(
   return onChangedOperation(writtenEventType, documentOrOpts, handler);
 }
 
+/**
+ * Event handler which triggers when a document is created, updated, or deleted in Firestore.
+ * This trigger will also provide the authentication context of the principal who triggered the event.
```

Comment:
Suggest "also provides"

Avoid "will" wherever you can.  Please check this globally in this PR, thanks!